modelina-website/scripts/build-examples.js

```diff
@@ -1,6 +1,12 @@
 const path = require('path');
 const {readdir, readFile, writeFile, mkdir, } = require('fs/promises');
 
+const refactorExampleName = (name) => {
+  const [first, ...rest] = name.split('-');
+  return first.charAt(0).toUpperCase() + first.slice(1) + " " + rest.join(" ")
+
+}
+
```

Comment:
what's the use of this code

Reply:
It's actually solving another issue #1417, I made a pull request before this for this change.

Reply:
well then make two different branches for both PR

Reply:
As @sambhavgupta0705 said, try not to depend two PRs with each other as much as possible 🙂 No problems this time, but if there was any issues in your last PR, this one would not be independently mergeable 😄